docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+ - You need to add certs, keys, and the master IP to the kubeconfig file:
+    - if using the firewall-only security option, set the apiserver this way:
+      - `kubectl config set-cluster $CLUSTER_NAME --server=http://$MASTER_IP --insecure-skip-tls-verify=true`
+    - otherwise, do this to set the apiserver ip, client certs, and user credentials.
+      - `kubectl config set-cluster $CLUSTER_NAME --certificate-authority=$CA_CERT --embed-certs=true --server=https://$MASTER_IP`
+      - `kubectl config set-credentials $CLUSTER_NAME --client-certificate=$CLI_CERT --client-key=$CLI_KEY --embed-certs=true --token=$TOKEN`
+    - Set your cluster as the default cluster to use:
+      - `kubectl config set-context $CLUSTER_NAME --cluster=$CLUSTER_NAME --user=admin`
+      - `kubectl config use-context $CONTEXT  --cluster=$CONTEXT`
+
+Next, make a kubeconfig file for the kubelets and kube-proxy.  There are a couple of options for how 
+many distinct files to make:
+  1. use the same credential as the admin
+    - simple
+    - harder to add finer grained permissions later
+  1. one token and kubeconfig file for all kubelets, one for all kube-proxy, one for admin.
+    - mirrors what is done on GCE today
+  1. different credentials for every kubelet, etc.
+    - we are working on this but all the pieces are not ready yet.
+
+You can make the files by copying the `$HOME/.kube/config`, by following the code
+in `cluster/gce/configure-vm.sh` or by using the following template:
+```
+apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+  user:
+    token: ${KUBELET_TOKEN}
+clusters:
+- name: local
+  cluster:
+    certificate-authority-data: ${CA_CERT_BASE64_ENCODED}
+contexts:
+- context:
+    cluster: local
+    user: kubelet
+  name: service-account-context
+current-context: service-account-context
+```
+Put the kubeconfig(s) on every node.  The examples later in this
+guide assume kubeconfigs in `/var/lib/kube-proxy/kubeconfig` and
+`/var/lib/kubeletproxy/kubeconfig`.
+
+#### Selecting Binaries
+You will need binaries for:
+  - etcd
+  - docker or rkt
+  - kubelet
+  - kube-proxy
+  - kube-apiserver
+  - kube-controller-manager
+  - kube-scheduler
+
+You can use a binary release (recommended) or build your own following the instructions in the
+[Developer Documentation]( ../devel/README.md).  Only using a binary release is covered in this guide.
+
+Download the [latest binary release](
+https://github.com/GoogleCloudPlatform/kubernetes/releases/latest) and unzip it.
+Then locate `./kubernetes/server/kubernetes-server-linux-amd64.targz` and unzip *that*.
+Then, within the second set of unzipped files, locate `./kubernetes/server/bin`, which contains
+all the necessary binaries.
+
+The kube-apiserver, kube-controller-manager, and kube-scheduler, you will run as pods,
+so there needs to be a docker image in whatever docker repository you are using.
+Options:
```

Comment:
rkt is not mentioned here. Are the rkt images in GCR too? Otherwise we should add where and how to get those too?